### PR TITLE
envycontrol: add page

### PR DESCRIPTION
**Pull request by rafay99-epic**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/envycontrol.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+`sudo envycontrol -s nvidia`
+
+- Switch to Integrated Graphic
+
+`sudo envycontrol -s integrated`
+
+- Switch to hybrid Graphic
+
+`sudo envycontrol -s hybrid`
```
Comment (kbdharun) [inline pages/linux/envycontrol.md:40]:
```suggestion
`sudo envycontrol -s {{nvidia|integrated|hybrid}}`
```

File: pages/linux/envycontrol.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+- Version
+
+`envycontrol -v`
+
+- Display Manager
+
+`envycontrol --dm`
+
+- Reset
```
Comment (kbdharun) [inline pages/linux/envycontrol.md:21]:
```suggestion
- Reset settings:

```

File: pages/linux/envycontrol.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+> Options:
+
+- Help 
+`envycontrol -h`
+
+- Version
+
+`envycontrol -v`
+
+- Display Manager
```
Comment (kbdharun) [inline pages/linux/envycontrol.md:17]:
```suggestion
- Specify your display manager manually:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/envycontrol.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- Specify your display manager manually:
+
+`envycontrol --dm`
+
+- Reset settings:
+
+`sudo envycontrol --reset`
+
+> Switch Graphic Mode:
+
```
Comment (kbdharun) [inline pages/linux/envycontrol.md:23]:
```suggestion
```

**Comment by rafay99-epic** [comment]
I have implemented all of the suggestion, But still not getting through the CI/CI(pull Request). Any Suggestions

**Comment by kbdharun** [comment]
> I have implemented all of the suggestion, But still not getting through the CI/CI(pull Request). Any Suggestions

Thanks, I have fixed it manually in my latest commit, in future when applying suggestions, go to the files tab, group them and make a single commit instead of creating multiple commits for each suggestion.

**Comment by rafay99-epic** [comment]
oh, I didn't know that, Thanks man

**Comment by rafay99-epic** [comment]
> LGTM, Thanks for your contribution.

Thank you for accepting this commit. I use EnvyControl and TLDRI on a daily basis; it is one of the best applications that I use.

**Comment by rafay99-epic** [comment]
This thread is ready to merge, Just required an Approval from blueskyson.

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @rafay99-epic 

sorry for the long wait time

**Comment by rafay99-epic** [comment]
It's alright... Thank you for merging my code..

**Comment by rafay99-epic** [comment]
> LGTM :) Thanks @rafay99-epic
> 
> sorry for the long wait time

Your welcome brother, It's alright; I'm happy to contribute to this amazing project.

**Pull request merged**